tests/backup_helper.go

```diff
@@ -3762,6 +3762,23 @@ func IsPresent(dataSlice interface{}, data interface{}) bool {
 	return false
 }
 
+// AppendList append only unique items of one list into another list
```

Comment:
How about this?
```
func AppendList[T comparable](dataSlice1 []T, dataSlice2 []T) []T {
	// Estimate the required size of the map
	totalLength := len(dataSlice1) + len(dataSlice2)
	map1 := make(map[T]bool, totalLength)
	list1 := make([]T, 0, totalLength) // Initialize list1 with the total capacity

	for _, element := range dataSlice1 {
		if !map1[element] {
			map1[element] = true
			list1 = append(list1, element)
		}
	}
	for _, element := range dataSlice2 {
		if !map1[element] {
			map1[element] = true
			list1 = append(list1, element)
		}
	}
	return list1
}
```

Reply:
For smaller number of data, I think this will not make too much of a difference
Also we reduced one for loop but added 2 if statement which will run each time in for loop
Let me know your thoughts @kshithijiyer-px

Reply:
I feel it's better to keep it optimised from day one we never know where all we'll end up using it. If we use it in scale tests this might become helpful there. I'll leave the decision to you if you wish to change it this way. or not.

Reply:
done